features/command_line/bisect.feature

```diff
@@ -47,59 +47,67 @@ Feature: Bisect
       """
 
   Scenario: Use `--bisect` flag to create a minimal repro case for the ordering dependency
-    When I run `rspec --seed 1234`
-    Then the output should contain "10 examples, 1 failure"
-    When I run `rspec --seed 1234 --bisect`
+    When I run `rspec --seed 9876`
```

Comment:
I changed seed number because when seed is 1234 calculator_1_spec is executed before calculator_20_spec